std/bitmanip.d

```diff
@@ -1536,20 +1561,22 @@ public:
         return _ptr[0 .. dim];
     }
 
+    ///
     @system unittest
     {
-        bool[] ba = [1,0,1,0,1];
+        // bit array with 10 elements
+        auto a = BitArray([1,0,1,0,1,0,1,0,1,0]);
+        size_t[] v = cast(size_t[]) a;
 
-        auto a = BitArray(ba);
-        void[] v = cast(void[]) a;
-
-        assert(v.length == a.dim * size_t.sizeof);
+        assert(v.length == a.length / size_t.sizeof);
+        assert(v.length == a.dim);
     }
 
     /***************************************
      * Support for unary operator ~ for $(D BitArray).
      */
-    BitArray opCom() const pure nothrow
+    BitArray opUnary(string op)() const pure nothrow
```

Comment:
Unfortunately we can't just remove `opCom` if we can avoid it. If possible with the current implementation of overloading in DMD, we should have both `opCom` and `opUnary` side by side for one release per the proposed deprecation process, and then deprecate `opCom`.